app/constants.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+class Constants {
+	constructor() {
+	}
+
+	static get CURRENT_RAID_ID() {
+		return "current";
+	}
+
+	static get UNDEFINED_END_TIME() {
+		return "unset";
+	}
+}
+
+module.exports = Constants
```

Comment:
Out of curiosity, if this is a class for constants, why have a class and not a const object?